### PR TITLE
[Docs] add Additional filters doc

### PR DESCRIPTION
**Pull request by tomerm-iguazio**
[ML-7690](https://iguazio.atlassian.net/browse/ML-7690)

[ML-7690]: https://iguazio.atlassian.net/browse/ML-7690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/feature-store/sources-targets.md [hunk 1]
```diff
@@ -150,6 +150,28 @@ feature_set.set_targets([])
...
 ```
 
+## Parquet source
+In ParquetSource, while reading a source, besides start_time and end_time,
+you can also use an additional_filter attribute on other columns in your source,
+which works similarly to the filtering functionality in pandas (based on pyarrow library).
+
+This can increase performance when reading large Parquet files.
+
+```
```
Comment (jond01) [inline docs/feature-store/sources-targets.md]:
```suggestion
```py
```

As well as in the other examples

This should also enforce formatting accordingly (`make fmt`/`make lint-docs`)

File: docs/feature-store/sources-targets.md [hunk 2]
```diff
@@ -150,6 +150,28 @@ feature_set.set_targets([])
 df = fs.ingest(feature_set, source=source)
 ```
 
+## Parquet source
```
Comment (jillnogold) [inline docs/feature-store/sources-targets.md]: move entire section just before S3 so it follows the order in the table at the start

File: docs/feature-store/sources-targets.md [hunk 3]
```diff
@@ -307,7 +330,7 @@ To configure, pass the `db_url` or overwrite the `MLRUN_SQL__URL` env var, in th
 
 You can pass the schema and the name of the table you want to create or the name of an existing table, for example:
 
-```
+```python
 target = SQLTarget(
     table_name="my_table",
     schema= {"id": string, "age": int, "time": pd.Timestamp, ...}
```
Comment (jond01) [inline docs/feature-store/sources-targets.md]: Missing `,`:
```suggestion
    schema= {"id": string, "age": int, "time": pd.Timestamp, ...},
```
Caught only now thanks to marking it as a Python code block

**Review by assaf758 — CHANGES_REQUESTED** [review]
Looks good in general.
A description of the discrepancy between pandas filtering and pyarrow/parquet (with regard to None / NaN is missing.

**Review by jillnogold — APPROVED** [review]

**Review by assaf758 — APPROVED** [review]

**Pull request merged**